apps/financial-aid/web-veita/src/components/NewUserModal/NewUserModal.tsx

```diff
@@ -83,7 +85,15 @@ const NewUserModal = ({
         onStaffCreated()
       })
     } catch (e) {
-      setState({ ...state, hasSubmitError: true })
+      setState({
+        ...state,
+        hasSubmitError: true,
```

Comment:
Do we need `hasSubmitError` ? Can't we just check if errorMessage if undefined? 

That is if we have it by default as undefined

Reply:
goood idea😍